ui/tests/pages/components/task-groups.js

```diff
@@ -0,0 +1,26 @@
+import { collection, clickable, text } from 'ember-cli-page-object';
+import { singularize } from 'ember-inflector';
+
+export default function (
+  selector = '[data-test-task-group]',
+  propKey = 'taskGroups'
+) {
+  const lookupKey = `${singularize(propKey)}For`;
+
+  return {
+    [propKey]: collection(selector, {
```

Comment:
`praise`: dynamically setting a property key on a JavaScript object is a pretty advanced move! Good on ya!

Reply:
I just adapted it from https://github.com/hashicorp/nomad/blob/main/ui/tests/pages/components/allocations.js 😅